src/shapes/ActiveSelection.spec.ts

```diff
@@ -55,4 +56,20 @@ describe('ActiveSelection', () => {
     selection.add(new FabricObject({ left: 50, top: 50, strokeWidth: 0 }));
     expect(selection.item(0).getCenterPoint()).toEqual({ x: 50, y: 50 });
   });
+
+  it('sets coords after attaching to canvas', () => {
```

Comment:
Isn't this a canvas test? is the canvas executing the new code no?

Reply:
true, I have no idea where it should belong